lib/resources/iis_app_pool.rb

```diff
@@ -0,0 +1,97 @@
+# encoding: utf-8
+# frozen_string_literal: true
+# check for web applications in IIS
+# Note: this is only supported in windows 2012 and later
+
+class IisAppPool < Inspec.resource(1)
+  name 'iis_app_pool'
+  desc 'Tests IIS application pool configuration on windows.'
+  example "
+    describe iis_app_pool('DefaultAppPool') do
+      it { should exist }
+      its('enable32bit') { should cmp 'True' }
+      its('runtime_version') { should eq 'v4.0' }
+      its('pipeline_mode') { should eq 'Integrated' }
+    end
+  "
+
+  def initialize(pool_name)
+    @pool_name = pool_name
+    @pool_path = "IIS:\\AppPools\\#{@pool_name}"
+    @cache = nil
+    @inspec = inspec
```

Comment:
You shouldn't need to put this in an instance variable. Methods within this resource should be able to simply call `inspec.command` and similar directly, without using `@inspec`

Reply:
Removed this.